### PR TITLE
fix: added right margin to facing issues btn on dashboad detail page

### PR DESCRIPTION
**Pull request by SagarRajput-7**
#### Screenshots

Before: 
--------
image

After: 
--------
Screenshot 2024-06-27 at 12 40 20 PM

### CODE REVIEW

File: frontend/src/container/NewDashboard/DashboardDescription/Description.styles.scss [hunk 1]
```diff
@@ -58,6 +58,7 @@
 		display: flex;
 		justify-content: space-between;
 		align-items: center;
+		padding-right: 16px;
```
Comment (vikrantgupta25) [inline frontend/src/container/NewDashboard/DashboardDescription/Description.styles.scss:61]: should we add padding or margin?

Comment (SagarRajput-7) [reply frontend/src/container/NewDashboard/DashboardDescription/Description.styles.scss:61]: I used padding as I didn't see the use of pushing the borders of the header inwards using margin, with padding the header's border will remain there and we will add space inside it.
also, the content below follows the same.

image

image

Comment (vikrantgupta25) [reply frontend/src/container/NewDashboard/DashboardDescription/Description.styles.scss:61]: didn't get you how is this causing any issues ? 
image

Comment (SagarRajput-7) [reply frontend/src/container/NewDashboard/DashboardDescription/Description.styles.scss]: Neither of them won't cause any issues, but I got your point about using margin to properly position the containers inside the containers.

**Review by vikrantgupta25 — APPROVED** [review]

File: frontend/src/container/NewDashboard/DashboardDescription/Description.styles.scss (hunk 1 above)
Comment (vikrantgupta25) [reply frontend/src/container/NewDashboard/DashboardDescription/Description.styles.scss]: https://mailchimp.com/resources/padding-vs-margin/ for more details

**Review by YounixM — APPROVED** [review]

**Pull request merged**